arrows/pdal/tests/test_pointcloud_io.cxx

```diff
@@ -34,19 +36,72 @@ class pointcloud_io : public ::testing::Test
 };
 
```

Comment:
You should also add a "create" test to confirm that you can construct this object from a string using the plugins.  See example here: https://github.com/Kitware/kwiver/blob/master/arrows/mvg/tests/test_integrate_depth_maps.cxx#L29-L37